data/campaigns/Winds_of_Fate/_main.cfg

```diff
@@ -19,7 +19,8 @@
     image="data/core/images/portraits/drakes/inferno.webp~SCALE(350,350)"
     description= _ "As the once plentiful hunting grounds of Morogor slip below the waves, war looms between drakes driven rogue by hunger and those still honorable. Yet the winds of fate may be shifting, for inspired by legends of vast far away lands, a pack of fledgling hunters makes ready to lead their kind on a journey towards salvation.
 
-" + _ "(Hard level, 11 scenarios.)"
+" + _ "(Hard level, 11 scenarios.)
+<i>Features full recall costs and no gold carryover.</i>"
```

Comment:
I'd prefer this to be a separate translatable string, rather than combining it with the difficultly and scenario count. That would allow it to be moved above the scenario count without invalidating the translation, or removed if @CelticMinstrel's suggestion of lesser recall costs is taken.